src/omv/modules/py_tf.c

```diff
@@ -619,12 +575,252 @@ STATIC mp_obj_t py_tf_segment(uint n_args, const mp_obj_t *pos_args, mp_map_t *k
 }
 STATIC MP_DEFINE_CONST_FUN_OBJ_KW(py_tf_segment_obj, 2, py_tf_segment);
 
+typedef enum {
+    PY_TF_MODEL_FOMO,
```

Comment:
```suggestion
    PY_TF_MODEL_FOMO =  31,
```
Number of ops, you can get the number of ops using:
```
tf.lite.experimental.Analyzer.analyze("model.tflite")
```
Max index + 1.

Reply:
This is now outdated, we should use the ops hash.

Reply:
Yup